release/test_update_bundles_yml.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
@lucasmiranda0, can you please rename the test file to `test_publishing.sh` instead?

This way, we can follow the existing pattern for test files names. :slightly_smiling_face: 

Thanks